ipwndfu

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python
```

Reply:
****

Reply:
The suggested change is identical to the current change.